src/lint/validators.nim

```diff
@@ -172,12 +172,22 @@ proc checkBoolean*(data: JsonNode; key, path: string; isRequired = true): bool =
   elif isRequired:
     result.setFalseAndPrint("Missing key: " & q(key), path)
 
-proc checkInteger*(data: JsonNode; key, path: string; isRequired = true): bool =
+proc checkInteger*(data: JsonNode; key, path: string; isRequired = true;
+                   allowed: Slice): bool =
```

Comment:
I've kept `allowed` as a required argument, for now. All of the integers in the spec so far have a restricted value:

> The "version" value must be the integer 3
> The "online_editor.indent_size" value must be a positive integer (>= 0)
> The "exercises.practice[].difficulty" value must be an integer >= 0 and <= 10

Reply:
I'm fine with that. We can update later if needed.